packages/@okta/vuepress-site/docs/reference/api/idps/index.md

```diff
@@ -455,6 +457,158 @@ For example: `?RelayState=:anyUrlEncodedValue`
 The deep link for the above three parts is:<br>
 `https://{myOktaDomain}.com/sso/saml2/:idpId/app/:app-location/:appId/sso/saml?RelayState=:anyUrlEncodedValue`
 
+#### Add Apple Identity Provider
+
+Adds a new `Apple` type IdP to your organization
+
+##### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "type": "APPLE",
+  "name": "Apple",
+  "protocol": {
+    "type": "OIDC",
+    "scopes": ["openid", "email", "name"],
+    "credentials": {
+      "client": {
+        "client_id": "your-client-id"
+      },
+      "signing": {
+        "privateKey": "MIGTAgEAMBM........Cb9PnybCnzDv+3cWSGWqpAIsQQZ",
+        "kid": "test key id",
+        "teamId": "test team id"
+      }
+    }
+  },
+  "policy": {
+    "provisioning": {
+      "action": "AUTO",
+      "profileMaster": true,
+      "groups": {
+        "action": "NONE"
+      },
+      "conditions": {
+        "deprovisioned": {
+          "action": "NONE"
+        },
+        "suspended": {
+          "action": "NONE"
+        }
+      }
+    },
+    "accountLink": {
+      "filter": null,
+      "action": "AUTO"
+    },
+    "subject": {
+      "userNameTemplate": {
+        "template": "idpuser.email",
+        "type": null
```

Comment:
do we want to include these null values in the example?

Reply:
removed